assets/css/global.less

```diff
@@ -874,6 +874,80 @@ a.caret-right{
   }
 }
 
+// Deployement guides
+.guide-card {
+  margin: 1.5rem;
+  max-height: 420px;
+  overflow : hidden;
+  text-overflow: ellipsis;
+  display: flex;
+}
+
+.card {
+  background: white;
+  padding: 6%;
+  color: black;
+  border-radius: 5px;
+  & h3, h2 {
+    color: black;
+  }
+  & a {
+    color: #06A6FD;
+    font-weight: 200;
+  }
+}
+
+.card-shadow{
```

Comment:
From the video, it looks like all of the parts of the post are in one, big, white box with a shadow. I think in Sam's designs, each top-level section is in a separate box with a shadow.

Reply:
Chiming in, adding new sections is not trivial, it would require adding another JS layer of parsing to jekyll's md and asciidoc parser if we want to insert custom html such as divs and classes around a whole section (which is different than adding a class just to the h2 or to individual p tags). Or it would require significant CSS-fu. I tried pairing with Hope yesterday where I attempted to use ::before for this, but we've limitatioins with that (we are allowed to style the ::before but the ::before content cannot contain any html, just pure text). Hope had some ideas she said she would try after our pairing, but given the nature of this and given how not final the design is, Id recommend against spending more time on this for now.